pmk/bootstrap-config.yaml

```diff
@@ -7,4 +7,11 @@ spec:
       disable: false # Set to false to enable creation of the default node role for managed machine pools
     fargate:
       disable: true # Set to false to enable creation of the default role for the fargate profiles
-
+  clusterAPIControllers: # GetPolicy permissions required for AWSManagedMachinePool
```

Comment:
Do we need a change in this file as well? i thought this was for legacy clusters

Reply:
Yes, this change is required, the template is generated using `pmk/bootstrap-config.yaml`